plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -852,12 +853,18 @@ class GesturesPluginTest {
       0.0,
       1.0
     )
+    val centerScreen = ScreenCoordinate(50.0, 50.0)
+    presenter.centerScreen = centerScreen
     val listener: OnShoveListener = mockk(relaxed = true)
     val gestureDetector = mockk<ShoveGestureDetector>(relaxUnitFun = true)
     presenter.addOnShoveListener(listener)
-    val result = presenter.handleShove(gestureDetector, 15.0f)
+    val result = presenter.handleShove(gestureDetector, -15.0f)
+    val expectedCameraOptions = cameraOptions {
+      pitch(2.500000022351742)
```

Comment:
Why it's such a weird value of pitch?

Reply:
Cause `optimizedPitch = 1.0 - (0.1 * (-15.0 + 0))` = 2.5 and simply took the value that got calculated, looks a bit strange yeah. I could potentially re-write it a bit I guess to compare doubles in a better manner.